core/ArchiveProcessor/Parameters.php

```diff
@@ -267,7 +272,11 @@ public function logStatusDebug()
 
     public function __toString()
     {
-        return "[idSite = {$this->getSite()->getId()}, period = {$this->getPeriod()->getLabel()} {$this->getPeriod()->getRangeString()}, segment = {$this->getSegment()->getString()}, plugin = {$this->getRequestedPlugin()}, report = {$this->getArchiveOnlyReport()}]";
+        $requestedReports = $this->getArchiveOnlyReport();
+        if (is_array($requestedReports)) {
+            $requestedReports = json_encode($requestedReports);
+        }
```

Comment:
Are there keys and values so that json_encode produces a better result or is it just values that can be imploded with commas?
```suggestion
        if (is_array($requestedReports)) {
            $requestedReports = implode(', ', $requestedReports);
        }
```